backend/api/v1/endpoints/Auth/auth.py

```
@@ -0,0 +1,101 @@
+    #     raise HTTPException(status_code=400, detail="This is not an educational email address.")
+    db_user = db.query(models.User).filter(models.User.username == user.username).first()
+    if db_user:
+        raise HTTPException(status_code=400, detail="Username already taken")
+    
+    hashed_password = hash_password(user.password)
+    new_user = models.User(
+        username=user.username,
+        email=user.email,
+        hashed_password=hashed_password,
+        profile_completed=False,  # ✅ Profile completion starts separately
+        is_verified=False  # ✅ Email verification status
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)